docs/ref/models/database-functions.txt

```diff
@@ -982,3 +982,545 @@ gaps.
+    >>> print(vector.x_exp, vector.y_exp)
+    (221.40641620418717, 0.1353352832366127)
+ 
+It can also be registered as a transform. For example::
+ 
+    >>> from django.db.models import FloatField
+    >>> from django.db.models.functions import Exp
+    >>> FloatField.register_lookup(Exp, 'exp')
+    >>> # Get vectors whose exp() greater than 10
+    >>> vectors = Vector.objects.filter(x__exp__gt=10, y__exp__gt=10)
+
+``Floor``
+---------
+
+.. class:: Floor(expression, **extra)
+
+Accepts a single numeric field or expression and returns the largest integer
+value not greater than expression.
+
+.. warning::
+
+    The behavior of ``Floor`` may be ``null`` if the input is null.
+
+Usage example::
+ 
+    >>> # Get the floor() of the coordinates
+    >>> from django.db.models.functions import Floor
+    >>> Vector.objects.create(x=5.4, y=-2.3)
+    >>> vector = Vector.objects.annotate(x_floor=Floor('x'), y_floor=Floor('y')).get()
+    >>> print(vector.x_floor, vector.y_floor)
+    (5, -3)
+ 
+It can also be registered as a transform. For example::
+ 
+    >>> from django.db.models import FloatField
+    >>> from django.db.models.functions import Floor
+    >>> FloatField.register_lookup(Floor, 'floor')
+    >>> # Get vectors whose floor() greater than 10
+    >>> vectors = Vector.objects.filter(x__floor__gt=10, y__floor__gt=10)
+
+``Log``
+--------
+
+.. class:: Log(expression, **extra)
+
+Accepts a single numeric field or expression and returns the natural logarithm of it.
+
+.. warning::
+
+    The behavior of ``Log`` may be ``null`` if the input is null or less than or
+    equal to 0.
+
+Usage example::
+ 
+    >>> # Get the log() of the coordinates
+    >>> from django.db.models.functions import Log
+    >>> Vector.objects.create(x=5.4, y=233.0)
+    >>> vector = Vector.objects.annotate(x_ln=Log('x'), y_ln=Log('y')).get()
+    >>> print(vector.x_ln, vector.y_ln)
+    (1.6863989535702288, 5.4510384535657)
+ 
+It can also be registered as a transform. For example::
+ 
+    >>> from django.db.models import FloatField
+    >>> from django.db.models.functions import Log
+    >>> FloatField.register_lookup(Log, 'log')
+    >>> # Get vectors whose value greater than e
+    >>> vectors = Vector.objects.filter(x__log__gt=1, y__log__gt=1)
+
+``Mod``
+--------
+
+.. class:: Mod(expression1, expression2, **extra)
+
+Modulo operation. Accepts two numeric fields or expression1 and expression2
+and returns the remainder of expression1 divided by expression2.
+
+.. warning::
+
+    The behavior of ``Mod`` may be ``null`` if the input is null or is 0.
+
+Usage example::
+ 
+    >>> # Get the mod() of the coordinates
+    >>> from django.db.models.functions import Mod
+    >>> Vector.objects.create(x=5.4, 2.3)
+    >>> vector = Vector.objects.annotate(x_mod=Mod('x'), y_mod=Mod('y')).get()
+    >>> print(vector.x_mod, vector.y_mod)
+    (0.8)
+ 
+``Pi``
+-------
+
+.. class:: Pi(**extra)
+
+Accepts zero arguments and returns value of pi. 
```

Comment:
`Returns the value of the mathematical constant Pi` sounds a bit better to me.

Reply:
Why not use the actual π symbol?